### ISSUE TITLE
Tile crashes on first use (permission already granted)

### ISSUE THREAD

**Issue by ataulm** [ataulm/talkback-toggle-tile#2]
- Create new Genymotion device, 7.0.0
- Install TalkBack 5.1.0
- Install TalkBack Toggle Tile
- set permission
- try to use tile -> crashes (expected: talkback enables, starts tutorial)

- enable talkback
- try to use tile -> nothing happens (expected: talkback turn off)

- disable talkback
- try to use tile -> works as expected :boom: 

what.

**Comment by ataulm** [comment]
Tried it on a real device. Unable to start fresh without TalkBack since it's preinstalled on new devices (rolled back to factory).

- rollback to factory TalkBack
- install TalkBack 5.1.0 (`adb install -r`)
- install talkback toggle tile
- set permission
- try using tile
- observe that talkback tutorial starts as expected

Tried again on Genymotion device (started by uninstalling TalkBack and uninstalling the tile), could not reproduce bug.
